pyproject.toml

```diff
@@ -28,8 +28,9 @@ dependencies = [
     # "roman_datamodels>=0.21.0,<0.22.0",
     "roman_datamodels @ git+https://github.com/spacetelescope/roman_datamodels.git",
     "scipy >=1.11",
-#    "stcal>=1.8.0,<1.9.0",
-    "stcal @ git+https://github.com/spacetelescope/stcal.git@main",
+    # "stcal>=1.8.0,<1.9.0",
+    # "stcal @ git+https://github.com/spacetelescope/stcal.git@main",
+    "stcal @ git+https://github.com/emolter/stcal.git@JP-3768",
```

Comment:
Temporarily set to source branch for https://github.com/spacetelescope/stcal/pull/292 for testing that (and this) PR.